### PR TITLE
feat(example): add sorry page - FRONT-2329

### PR DESCRIPTION
**Pull request by emeryro**
preview: https://refined-github-html-preview.kidonng.workers.dev/ec-europa/europa-component-library/raw/feat/FRONT-2329-sorry-page/examples/sorry-page/sorry.html
HTML code: https://raw.githubusercontent.com/ec-europa/europa-component-library/feat/FRONT-2329-sorry-page/examples/sorry-page/sorry.html

note:
- this page is using resources from the cdn (css, images, icons), to keep it ready to use on any environment
- to be seen if this page has to be showcased on ECL website/playground

### CODE REVIEW

**Review by papegaill — COMMENTED** [review]
Nice, my only remark would be the alignments for mobile, the difference in margin between the elements and the bottom border which doesn't go all the way, gives the impression that it is a little broken?

Screenshot 2021-06-15 at 12 21 29

Here is a test where the gray border goes to 100% and the spacing between the items is equal: https://codepen.io/bypapegaill/pen/bGqORZm
I also reduced the padding between the title of the items and the text. Added spacing to the columns to give more visual space in the middle in desktop format.

what do you think?

**Comment by emeryro** [comment]
there are some spacing specified by designers here https://citnet.tech.ec.europa.eu/CITnet/confluence/pages/viewpage.action?spaceKey=ECL&title=01.+Sorry+Page
So these ones have to be respected.
But I'll apply some of your recommendations, thanks for that

File: examples/sorry-page/sorry.html [hunk 1]
```diff
@@ -0,0 +1,1001 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="ecl-u-bg-blue-100 ecl-u-pa-l"></footer>
+
+    <script
+      src="https://cdn1.fpfis.tech.ec.europa.eu/ecl/v2.38.0/eu-preset-website/scripts/ecl-eu-preset-website.js"
```
Comment (papegaill) [inline examples/sorry-page/sorry.html]: yet another question? Is it really necessary to have the ECL preset JS loaded for this page, as well as `autoInit`? `svgforeverybody` isn't it enough?

Comment (emeryro) [reply examples/sorry-page/sorry.html]: fair point, I'll remove it

**Review by papegaill — APPROVED** [review]

**Pull request merged**